src/components/ProductForm/ProductForm.tsx

```diff
@@ -0,0 +1,156 @@
+import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
+import { ReactElement } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+import { Dropdown } from 'components/Dropdown/Dropdown';
+import { useProductFormContext } from './ProductFormContext';
+import { Button } from 'components/Buttons/Button';
+
+export const ProductForm = (): ReactElement => {
+  const formContext = useProductFormContext();
+  const [formProduct, setFormProduct] = formContext.useFormProduct;
+
+  //Placeholder Data
+  interface ITag {
+    id: number;
+    name: string;
+  }
+  const allTags: ITag[] = [
+    { id: 0, name: 'Vegan' },
+    { id: 1, name: 'Vegetarian' },
+    { id: 2, name: 'Good Food' },
+    { id: 3, name: 'Nut-Free' },
+    { id: 4, name: 'Gluten-Free' },
+    { id: 5, name: 'Breakfast Menu' },
+    { id: 6, name: 'Vegan' },
+    { id: 7, name: 'Vegetarian' },
+    { id: 8, name: 'Good Food' },
+    { id: 9, name: 'Nut-Free' },
+    { id: 10, name: 'Gluten-Free' },
+    { id: 11, name: 'Breakfast Menu' },
+    { id: 12, name: 'Vegan' },
+    { id: 13, name: 'Vegetarian' },
+    { id: 14, name: 'Good Food' },
+    { id: 15, name: 'Nut-Free' },
+    { id: 16, name: 'Gluten-Free' },
+    { id: 17, name: 'Breakfast Menu' },
+    { id: 18, name: 'Vegan' },
+    { id: 19, name: 'Vegetarian' },
+    { id: 20, name: 'Good Food' },
+  ];
+  const categories = ['Mains', 'Sides', 'Drinks'];
+
+  const toggleTag = (event: React.ChangeEvent<HTMLInputElement>, toggleTagId: number): void => {
+    if (event.target.checked) {
+      setFormProduct({
+        ...formProduct,
+        tags: formProduct.tags.concat([toggleTagId]),
+      });
+    } else {
+      setFormProduct({
+        ...formProduct,
+        tags: formProduct.tags.filter((tagId) => tagId !== toggleTagId),
+      });
+    }
+  };
+
+  return (
+    <>
+      <form className="w-full overflow-auto">
+        <div className="grid grid-cols-4 gap-4 p-2">
+          <div className="col-span-2 row-span-3 flex h-full w-full items-center justify-center bg-gray-200">
+            <FontAwesomeIcon icon={faPen} />
+          </div>
+          <div className="col-span-2">
```

Comment:
if you add the text colour to the parent div, it should add it to all the children too. Get the text colour from figma

Reply:
the colours of the inputs and the titles are different so all the inputs are inheriting black text with the titles specified as grey-400. 
![details - Copy](https://user-images.githubusercontent.com/116075166/217798142-3e9a6231-f31b-475d-8a93-3d3bf0e498b0.png)